test-app/tests/integration/components/bs-modal/footer-test.gts

```diff
@@ -37,15 +40,18 @@ module('Integration | Component | bs-modal/footer', function (hooks) {
     assert.strictEqual(close.callCount, 1, '@onClose is called exactly once');
   });
 
+  // TODO The logs and expected values in the assertions for the button type do not agree on the expected button type.
```

Comment:
Is this something we still need to do? Or has it been fixed?

Reply:
This one is solved, I just forgot to remove the todo